dev/tests/bundle/tasks.js

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
In the dev tasks we don't need 100%. Covering the most important paths will be enough.